init.lua

```diff
@@ -59,19 +59,34 @@ end
 
 -- Usage:
 --   resizer = SkyRocket:new({
+--     opacity = 0.3,
 --     moveModifiers = {'cmd', 'shift'},
+--     moveMouseButton = 'left',
 --     resizeModifiers = {'ctrl', 'shift'}
+--     resizeMouseButton = 'left',
 --   })
 --
+local function buttonNameToEventType(name)
+    if name == 'left' then
+        return hs.eventtap.event.types.leftMouseDown
+    end
+    if name == 'right' then
+        return hs.eventtap.event.types.rightMouseDown
+    end
+    error('only "left" and "right" mouse button supported, got ' .. name)
+end
+
 function SkyRocket:new(options)
   options = options or {}
 
   local resizer = {
     disabledApps = tableToMap(options.disabledApps or {}),
     dragging = false,
     dragType = nil,
+    moveStartMouseEvent = buttonNameToEventType(options.moveMouseButton or 'left'),
```

Comment:
**praise** Thanks for keeping the defaults the same and not breaking past users.